typhos/app.py

```diff
@@ -27,6 +27,8 @@ def launch_suite(suite):
     """Creates a main window and execs the application."""
     window = QMainWindow()
     window.setCentralWidget(suite)
+    window.setWindowTitle(suite.windowTitle())
+    window.setUnifiedTitleAndToolBarOnMac(True)
```

Comment:
I'm surprised to see such a specifically named function in pyqt. Gave me a chuckle.